omod/src/main/java/org/openmrs/module/coreapps/page/controller/ActiveVisitsPageController.java

```diff
@@ -42,10 +42,9 @@ public String get(UiSessionContext sessionContext, PageModel model, @SpringBean
         if (sessionLocation == null) {
             return "redirect:login.htm";
         }
-		Location visitLocation = null;
-		if (sessionLocation != null) {
-			visitLocation = service.getLocationThatSupportsVisits(sessionLocation);
-		}
+
+        Location visitLocation = service.getLocationThatSupportsVisits(sessionLocation);
```

Comment:
Shouldn't there be two tabs instead of eight spaces at this point?

Reply:
Yeah, you right... I corrected it